examples/adc-continious-dma.rs

```diff
@@ -47,7 +49,10 @@ fn main() -> ! {
     let pa0 = gpioa.pa0.into_analog();
 
     info!("Setup Adc1");
-    let mut delay = cp.SYST.delay(&rcc.clocks);
+    // let mut delay = cp.SYST.delay(&rcc.clocks);
+    let mut delay = DelayFromCountDownTimer::new(
+        Timer::new(dp.TIM6, &rcc.clocks).start_count_down(100u32.millis()),
+    );
```

Comment:
Why is this needed? :)

Reply:
I switched out the SYST delay provider because cortex-m doesn't implement the new delay trait yet, it's probably not necessary though

Reply:
Is there any way to avoid this since I would imagine this would break a lot of users code?

Reply:
Using the old e-hal 0.2 trait should still work, but the function names are in conflict with the new ones which are exported in the prelude

Reply:
Might be a good idea to see how/if this is solved in for example [stm32-rs/stm32f4xx-hal](https://github.com/stm32-rs/stm32f4xx-hal)

Reply:
DelayNs has been implemented in https://github.com/rust-embedded/cortex-m/commit/ec6b3ba49db60b23885921d238d3a9e86cf3e9e9 but it hasn't been included in a release yet